gaffer-as-a-service/gaas-rest/src/test/java/uk/gov/gchq/gaffer/gaas/integrationtests/GraphControllerIT.java

```diff
@@ -207,7 +207,7 @@ public void createGraph_shouldReturn500BadRequestWhenStoreTypeIsInvalidType() th
                 .content(inputJson)).andReturn();
 
         assertEquals(500, result.getResponse().getStatus());
-        final String expected = "{\"title\":\"RuntimeException\",\"detail\":\"StoreType is Invalid must be defined Valid Store Types supported are: accumuloStore, federatedStore, mapStore, proxyStore\"}";
+        final String expected = "{\"title\":\"RuntimeException\",\"detail\":\"StoreType is Invalid must be defined Valid Store Types supported are: proxyStore, mapStore, accumuloStore, federatedStore\"}";
```

Comment:
Is this caused by the store types being output in a different order for different users? May want something more permanent if so. Could set `expected` to be a regular expression that captures the names. You can then split, order and compare to an `expectedStoreTypeList`.

Reply:
@r32575 please review again I made the changes as you requested.